src/steps/merge_branch_step.go

```diff
@@ -9,6 +9,8 @@ import (
 type MergeBranchStep struct {
 	NoOpStep
 	BranchName string
+
```

Comment:
I'm trying to understand when you add empty lines and when not. I'm tending more towards a